### PR TITLE
Added reveal / preview

### PR DESCRIPTION
**Pull request by rbui**
also some asset stuff

### CODE REVIEW

File: VouchrSdkShowCase-iOS/HomeViewController.m [hunk 1]
```diff
@@ -128,6 +128,16 @@ - (void)setupVouchrEngine {
         // override any default managers here
     }];
     
+    VoucherRevealManager *revealManager = [VoucherRevealManager voucherRevealManagerWithBuilder:^(VoucherRevealManagerBuilder *builder) {
+        VouchrTheme *theme = [self surpriiseTheme];
+        theme.displayFontTheme.color = UIColor.blackColor;
+        builder.revealViewControllerTheme = theme;
+        //builder.shouldPlayGameFirst = NO;
```
Comment (alicehuanggg) [inline VouchrSdkShowCase-iOS/HomeViewController.m]: whats dis

File: VouchrSdkShowCase-iOS/HomeViewController.m [hunk 2]
```diff
@@ -136,6 +146,8 @@ - (void)setupVouchrEngine {
         builder.youtubeManager = [[VouchrEngine instance] youtubeManager];
         builder.gameManager = [[VouchrEngine instance] gameManager];
         builder.merchantManager = [[VouchrEngine instance] merchantManager];
+        builder.revealFlowCoordinator = revealFlowCoordinator;
+
 //        builder.fullScreenLoadingView = [[SurpriiseLoadingView alloc] initWithFrame:CGRectZero];
```
Comment (alicehuanggg) [inline VouchrSdkShowCase-iOS/HomeViewController.m]: and dis?

File: VouchrSdkShowCase-iOS/HomeViewController.m (hunk 1 above)
Comment (rbui) [reply VouchrSdkShowCase-iOS/HomeViewController.m]: Extra config options that i don't want us to forget about

File: VouchrSdkShowCase-iOS/HomeViewController.m (hunk 2 above)
Comment (rbui) [reply VouchrSdkShowCase-iOS/HomeViewController.m]: Extra config options that i don't want us to forget about

File: VouchrSdkShowCase-iOS/HomeViewController.m (hunk 1 above)
Comment (alicehuanggg) [reply VouchrSdkShowCase-iOS/HomeViewController.m]: maybe add a comment to clarify?

File: VouchrSdkShowCase-iOS/HomeViewController.m (hunk 2 above)
Comment (alicehuanggg) [reply VouchrSdkShowCase-iOS/HomeViewController.m]: maybe add a comment to clarify?

**Pull request merged**